INSTUI-2873 New contribution process documentation (#410)
instructure/instructure-ui: docs/getting-started/contributing.md

Contributions are welcome! If you want to submit a change you will need to open a pull request on Github. To do so first fork the repository to your own Github account.

### What can you contribute?

> If you found a bug or want to add a minor feature, go ahead! If it's a larger thing (like a new component), read on.

Reviewer: ```suggestion
If you found a bug or want to add a minor feature, go ahead!

If you'd like to add a new component, or change/add new visuals, please read on.
```

We try to keep the InstUI easy to use, easy to learn and free from bloat.
For this reason we will only add new components if it fits into the current design patterns, and the new component cannot be
constructed from the existing ones. For these reasons adding a new component is the following process:

> - You should write a document what do you want to add and what use case does it solve.

Reviewer 2: ```suggestion
- You should write a document which includes the component description and the use cases it's meant to be used in.
```

> - You should make some kind of design sketch that documents the design of the pattern (e.g. with [Figma](https://www.figma.com/)).

Reviewer 2: ```suggestion
- You should make some kind of design sketch that documents the design of the component (e.g. with [Figma](https://www.figma.com/)).
```

- Our design team will review and approve this design sketch.

> - If the sketch is OK, you will need to make a pull request (read on for our code guidelines), the InstUI dev team and our a11y team will review the code and merge it.

Reviewer: ```suggestion
- If the sketch is OK, you will need to make a pull request (read on for our code guidelines).
- The InstUI dev team and our a11y team will review the code and merge it.
```

### Building InstUI from the source

> Follow these steps to download and compile Instructure-UI on your machine:

Reviewer 2: ```suggestion
Follow these steps to download and compile Instructure UI on your machine:
```
